CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## Release 1.7.2
+
+Highlights:
+- eBPF tracker working on GKE
+- Upgrade kubernetes api to kubernetes-1.9.1
+
+New features and enhancements:
+- vendor: bump tcptracer-bpf
+	[#3042](https://github.com/weaveworks/scope/pull/3042)
+
+Bug fixes and minor improvements:
+- Close terminal pipe, when closing the pod panel
+	[#3045](https://github.com/weaveworks/scope/pull/3045)
+- Fetch cronjobs from 'batch/v1beta1'
+	[#3044](https://github.com/weaveworks/scope/pull/3044)
```

Comment:
This fixes a bug which caused CronJobs in recent k8s not to appear in Scope, correct? Worth highlighting that.